SmartDeviceLink/SDLRTPH264Packetizer.h

```diff
@@ -0,0 +1,47 @@
+//
+//  SDLRTPH264Packetizer.h
+//  SmartDeviceLink-iOS
+//
+//  Created by Sho Amano on 4/11/17.
+//  Copyright © 2017 Xevo Inc. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "SDLH264Packetizer.h"
+
+@interface SDLRTPH264Packetizer : NSObject <SDLH264Packetizer>
+
+/**
+ * Payload Type (PT) of RTP header field.
+ *
+ * @note This must be between 0 and 127. Default value is 96.
+ */
+@property (assign, nonatomic) UInt8 payloadType;
```

Comment:
I've put some documentation with commit https://github.com/smartdevicelink/sdl_ios/pull/703/commits/d585057de0ecd76efeaf93a783fb3617b0ffabfc. Hope this will make clearer.